static/app/views/settings/projectMetrics/metricsExtractionRuleForm.tsx

```diff
@@ -478,9 +478,9 @@ export function MetricsExtractionRuleForm({
                   </Tooltip>
                 )}
                 <TooltipIconLabel
-                  label={t('Group and filter by')}
+                  label={t('Group by')}
                   help={t(
-                    'Select the tags that can be used to group and filter the metric. Tag values have to be non-numeric.'
+                    'Select the tags that can be used to group the metric. Tag values have to be non-numeric.'
```

Comment:
@ale-cota I noticed that some other copies also needed updating based on the new ones. Could you please confirm if everything is fine?

Reply:
It's actually still true that you can group and filter by these tags, so we can leave it in the description. We just didn't want it in the name, since then it looks redundant.